e2e/pages/launch_vscode.page.js

```diff
@@ -0,0 +1,45 @@
+// launch_vscode.page.js
+const { _electron: electron } = require('playwright');
+const path = require('path');
+const fs = require('fs');
+
+class LaunchVSCodePage {
+  constructor() {
+    this.vscodeApp = null;
+    this.window = null;
+  }
+
+  async launchVSCode(executablePath) {
+    this.vscodeApp = await electron.launch({
+      executablePath: executablePath, // Path to VSCode executable
+    });
+
+  }
```

Comment:
Since an instance cannot be constructed without actually opening vscode, we could just make "launchVScode" static and return a class instance from there.